src/en/components/link/base.md

```diff
@@ -0,0 +1,22 @@
+---
+github: https://github.com/cds-snc/gcds-components/tree/main/packages/web/src/components/gcds-link
+figma: https://www.figma.com/file/mh2maMG2NBtk41k1O1UGHV/Canadian-Digital-Service%E2%80%A8---GC-Design-System?type=design&node-id=2646-8407&mode=design&t=Fc8G98RMn9FnseDi-0
+permalink: false
+tags: ['linkEN', 'header']
+---
+
+# Link <br>`<gcds-link>`
+
+_Also called: anchor, hyperlink, call to action._
+
+A link is a navigational element that brings a person to a new page, website, file, or section on the current page.
+
+{% docLinks locale stage figma github %}
+{% enddocLinks %}
+
+{% componentPreview "Link component preview" %}
+<gcds-link class="me-400" href="#">Link</gcds-link>
+<gcds-link class="me-400" href="long-filename.pdf" download="file.pdf" type="application/pdf">Link</gcds-link>
+<gcds-link class="me-400" href="mailto:test@test.com?subject=Test%20Email">Link</gcds-link>
+<gcds-link class="me-400" href="tel:1234567890">Link</gcds-link>
+{% endcomponentPreview %}
```

Comment:
Are we intentionally skipping out the preview for an external link?

Reply:
Yes, because there are some debates around having the external icons or not ;)